### PR TITLE
Add scripts for Bear

### PR DESCRIPTION
**Pull request by tanaynistala**
## Description

Adds 6 new script commands to work with the [Bear](https://bear.app) note-taking app:

- Add to Bear: Create a new note, either through an input or from clipboard (enter `yes` exactly to use clipboard)
- Open Note: Open the note with the specified title
- Open Tag: Open the tag with the specified name
- Search Bear: Search all notes in Bear by term and/or tag
- Open Today: Open the Today section in Bear with an optional search query
- Open Todo: Open the Todo section in Bear with an optional search query

## Type of change

- [x] New script command

## Screenshot

image

## Dependencies / Requirements

Requires the [Bear](https://apps.apple.com/us/app/bear-beautiful-writing-app/id1091189122) app to be installed.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/bear/bear-open-tag.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# @raycast.authorURL https://github.com/tanaynistala
+# @raycast.description Open the specified tag in Bear.
+#
+# @raycast.icon images/bear-light.png
+# @raycast.iconDark images/bear-dark.png
+#
+# @raycast.mode silent
+# @raycast.packageName Bear
+# @raycast.schemaVersion 1
+# @raycast.argument1 { "type": "text", "placeholder": "Tag" }
```
Comment (unnamedd) [inline commands/apps/bear/bear-open-tag.sh:14]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Tag", "percentEncoded": true}
```

File: commands/apps/bear/bear-open-tag.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+open "bear://x-callback-url/open-tag?name=${1// /%20}"
```
Comment (unnamedd) [inline commands/apps/bear/bear-open-tag.sh:16]:
```suggestion
open "bear://x-callback-url/open-tag?name=${1}"
```

File: commands/apps/bear/bear-search.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# @raycast.title Search Bear
```
Comment (unnamedd) [inline commands/apps/bear/bear-search.sh:3]:
```suggestion
# @raycast.title Search
```

File: commands/apps/bear/bear-search.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# @raycast.description Search notes by keyword and/or tag in Bear.
+#
+# @raycast.icon images/bear-light.png
+# @raycast.iconDark images/bear-dark.png
+#
+# @raycast.mode silent
+# @raycast.packageName Bear
+# @raycast.schemaVersion 1
+# @raycast.argument1 { "type": "text", "placeholder": "Term", "optional": true }
+# @raycast.argument2 { "type": "text", "placeholder": "Tag", "optional": true }
```
Comment (unnamedd) [inline commands/apps/bear/bear-search.sh:15]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Term", "optional": true, "percentEncoded": true}
# @raycast.argument2 { "type": "text", "placeholder": "Tag", "optional": true, "percentEncoded": true}
```

File: commands/apps/bear/bear-search.sh [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+open "bear://x-callback-url/search?term=${1// /%20}&tag=${2// /%20}"
```
Comment (unnamedd) [inline commands/apps/bear/bear-search.sh:17]:
```suggestion
open "bear://x-callback-url/search?term=${1}&tag=${2}"
```

File: commands/apps/bear/bear-today.sh [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+# @raycast.authorURL https://github.com/tanaynistala
+# @raycast.description Open the Today section in Bear.
+#
+# @raycast.icon images/bear-light.png
+# @raycast.iconDark images/bear-dark.png
+#
+# @raycast.mode silent
+# @raycast.packageName Bear
+# @raycast.schemaVersion 1
+# @raycast.argument1 { "type": "text", "placeholder": "Search Query", "optional": true }
```
Comment (unnamedd) [inline commands/apps/bear/bear-today.sh:14]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Search Query", "optional": true, "percentEncoded": true}
```

File: commands/apps/bear/bear-today.sh [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+open "bear://x-callback-url/today?search=${1// /%20}"
```
Comment (unnamedd) [inline commands/apps/bear/bear-today.sh:16]:
```suggestion
open "bear://x-callback-url/today?search=${1}"
```

File: commands/apps/bear/bear-todo.sh [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+# @raycast.authorURL https://github.com/tanaynistala
+# @raycast.description Open the Todo section in Bear.
+#
+# @raycast.icon images/bear-light.png
+# @raycast.iconDark images/bear-dark.png
+#
+# @raycast.mode silent
+# @raycast.packageName Bear
+# @raycast.schemaVersion 1
+# @raycast.argument1 { "type": "text", "placeholder": "Search Query", "optional": true }
```
Comment (unnamedd) [inline commands/apps/bear/bear-todo.sh:14]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Search Query", "optional": true, "percentEncoded": true}
```

File: commands/apps/bear/bear-todo.sh [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+open "bear://x-callback-url/todo?search=${1// /%20}"
```
Comment (unnamedd) [inline commands/apps/bear/bear-todo.sh:16]:
```suggestion
open "bear://x-callback-url/todo?search=${1}"
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @tanaynistala, thanks for your contribution and to bring Bear to our community. 
Even though I wrote many comments, all of them almost follow the same line of thought.  

Waiting for you next interation! 🤘

**Comment by tanaynistala** [comment]
Thanks for the suggestions, @unnamedd! I've applied your suggested changes to this iteration.

**Pull request merged**